package.json

```diff
@@ -58,15 +59,16 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-test-loader": "^3.0.0",
-    "ember-cli-typescript": "^1.5.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-fetch": "^8.1.0",
     "ember-in-element-polyfill": "^1.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
+    "ember-page-title": "^6.2.2",
```

Comment:
Do we need this? Was it just brought in via an `ember init`?

Reply:
@rwjblue ember-page-title is used in the dummy app, embroider detected the missing dependency. Alternatively, I could remove the usage & the dependency now that we know it works.

Reply:
i've removed this